### ISSUE TITLE
Make it possible to customize individual fieldset fields

### ISSUE THREAD

**Issue by aerni** [statamic/ideas#1265]
I often find myself wanting to edit a few fields in a fairly complex fieldset. Mainly changing things like a field's title or instructions.

I believe the best option would be to make linked fieldsets configurable the same way you can configure individually linked fields. This would allow users to link complex fieldsets and change the config of individual fields if needed.

A simpler approach might be to add a new button that "explodes"  a linked fieldset into the blueprint. This button would convert the linked fieldset to actual blueprint fields.

![CleanShot 2024-11-15 at 13 00 36@2x](https://github.com/user-attachments/assets/e2f60dba-2629-4293-8b83-f21ab6c34008)

**Comment by adampatterson** [comment]
I agree, I've got a couple of fieldsets that include images and depending on the content collection, I need to save those images in different asset containers.

Maybe as a bonus, include a quick jump to editing the field set.
